src/main/java/com/endava/soa_5ed/task3_oop/Cat.java

```diff
@@ -0,0 +1,84 @@
+package com.endava.soa_5ed.task3_oop;
+
+public class Cat extends Feline implements Tail {
+    private double size;
+    private String name;
+
+    public Cat(String gender, String furColor, String name) {
+        super(gender, furColor);
+        this.name = name;
+        System.out.println("############ Creating Cat ############");
```

Comment:
Looks like a way of logging, dont you want to try log4j for this purpose?

Reply:
done, thank you for your suggestion